lib/plug/quiet_logger.ex

```diff
@@ -8,9 +8,18 @@ defmodule Plug.QuietLogger do
     %{log: log, path: path}
   end
 
+  def call(%{request_path: path} = conn, %{log: :info, path: paths}) when is_list(paths) do
```

Comment:
The hardcoded `:info` would prevent the configurability of the log level. Is that intended?